interface/super/edit_globals.php

```diff
@@ -573,6 +573,35 @@ function validate_file(){
         "<input type='button' value='Default' onclick=\"document.forms[0].form_$i.color.fromString('" . attr($flddef) . "')\">\n";
     }
 
+    else if ($fldtype == 'visit_category') {
+        $sql = "SELECT pc_catid, pc_catname, pc_cattype 
+                FROM openemr_postcalendar_categories
+                WHERE pc_active = 1 ORDER BY pc_seq";
+        $result = sqlStatement($sql);
+        echo "<select name=\"form_{$i}\" id=\"form_{$i}\">\n";
+        echo "<option value='_blank'>" . xl('None') . "</option>";
+        while ($row = sqlFetchArray($result)) {
+            $id = $row['pc_catid'];
+            $name = $row['pc_catname'];
+            if ($id < 9 && $id != "5") {
+                continue;
+            }
+
+            if ($row['pc_cattype'] == 3 && !$GLOBALS['enable_group_therapy']) {
+                continue;
+            }
+
+            $optionStr = '<option value="%pc_catid%"%selected%>%pc_catname%</option>';
+            $optionStr = str_replace("%pc_catid%", attr($id), $optionStr);
+            $optionStr = str_replace("%pc_catname%", xl($name), $optionStr);
```

Comment:
need to use:
``````
text(xl_appt_category($name))
``````
(this is a translation function that only works when user sets in globals to translate category names, which is on by default)